test/client/getOrderbook.ts

```diff
@@ -1,44 +1,27 @@
-// import BigNumber from "bignumber.js";
-// import { assert } from "chai";
-
+import BigNumber from 'bignumber.js'
 import { assert } from 'chai'
+
 import { BookOffersRequest } from '../../src'
+import { OfferLedgerFlags } from '../../src/models/ledger/offer'
 import requests from '../fixtures/requests'
 import responses from '../fixtures/responses'
 import rippled from '../fixtures/rippled'
 import { setupClient, teardownClient } from '../setupClient'
-import { addressTests } from '../testUtils'
-
-// function checkSortingOfOrders(orders) {
-//   let previousRate = "0";
-//   for (let i = 0; i < orders.length; i++) {
-//     const order = orders[i];
-//     let rate;
-
-//     // We calculate the quality of output/input here as a test.
-//     // This won't hold in general because when output and input amounts get tiny,
-//     // the quality can differ significantly. However, the offer stays in the
-//     // order book where it was originally placed. It would be more consistent
-//     // to check the quality from the offer book, but for the test data set,
-//     // this calculation holds.
+import { assertResultMatch, assertRejects } from '../testUtils'
 
-//     if (order.specification.direction === "buy") {
-//       rate = new BigNumber(order.specification.quantity.value)
-//         .dividedBy(order.specification.totalPrice.value)
-//         .toString();
-//     } else {
-//       rate = new BigNumber(order.specification.totalPrice.value)
-//         .dividedBy(order.specification.quantity.value)
-//         .toString();
-//     }
-//     assert(
-//       new BigNumber(rate).isGreaterThanOrEqualTo(previousRate),
-//       `Rates must be sorted from least to greatest: ${rate} should be >= ${previousRate}`
-//     );
-//     previousRate = rate;
-//   }
-//   return true;
-// }
+function checkSortingOfOrders(orders): boolean {
+  let previousRate = '0'
+  for (const order of orders) {
+    assert(
+      new BigNumber(order.quality).isGreaterThanOrEqualTo(previousRate),
+      `Rates must be sorted from least to greatest: ${
+        order.quality as number
+      } should be >= ${previousRate}`,
+    )
+    previousRate = order.quality
+  }
+  return true
```

Comment:
Is this return used? (Since this only returns if it doesn't throw an error, it seems like you could get the same effect without returning anything)